### ISSUE TITLE
Chemical formula for Drink Mix

### ISSUE THREAD

**Issue by emily-phet** [phetsims/molarity#86]
Description for Drink Mix under H2: Beaker needs to read:

"Drink Mix contains mainly citric acid, which has a chemical formula of C(sub)6H(sub)8O(sub)7."

Instead of "Chemical formula for Drink Mix is Drink Mix."

**Comment by twant** [comment]
@Matthew-phet @emily-phet Assigning these back over to you for review.

**Comment by twant** [comment]
As @zepumph was reviewing this, he mentioned that perhaps we'd like to add the chemical formula for citric acid as Drink Mix's chemical formula in other non-a11y parts of the sim. In the code, this caused a conditional, and as a result it might help with maintainability. @emily-phet what do you think?

**Comment by pixelzoom** [comment]
> As @zepumph was reviewing this, he mentioned that perhaps we'd like to add the chemical formula for citric acid as Drink Mix's chemical formula in other non-a11y parts of the sim. In the code, this caused a conditional, and as a result it might help with maintainability. @emily-phet what do you think?

I don't see any problem with maintainability.   And this is a question for the lead designer, @arouinfar.   

I'm skeptical that we'd want "Drink mix" to be changed to "Drink mix (C<sub>6</sub>H<sub>8</sub>O<sub>7</sub>)" on the beaker, because it was supposed to be a more "approachable".   If we do change it, consider that "Drink mix" also appears in Beer's Law Lab and Concentration.

**Comment by arouinfar** [comment]
@emily-phet I want to push back a bit on citric acid for a few reasons.
* The primary ingredient in powdered drink mixes such as Kool Aid or Gatorade is sucrose. (Admittedly, citric acid is the primary ingredient in zero-calorie drink mixes, but then we should probably call it "Diet Drink Mix" instead.)
* The teacher tips state: "Drink mix is assumed to have the same solubility as sucrose."
* An aqueous solution of citric acid saturates at [3.08 M (20 °C)](https://pubchem.ncbi.nlm.nih.gov/compound/Citric-acid#section=Solubility) and sucrose saturates at [6.13 M (25 °C)](https://pubchem.ncbi.nlm.nih.gov/compound/Sucrose#section=Solubility). In the sim, Drink Mix can have a concentration of up to 5.0 M without saturating, which would be consistent with sucrose.

> I'm skeptical that we'd want "Drink mix" to be changed to "Drink mix (C6H8O7)" on the beaker, because it was supposed to be a more "approachable" solute. And FYI, "Drink mix" also appears in Beer's Law Lab and Concentration.

I was not a part of the original design team for these sims, but this has also been my understanding @pixelzoom. By design, the chemical formula of Drink Mix was not shown, so I would be hesitant to display its chemical formula.

I find it a bit surprising that the chemical formula of Drink Mix is revealed in the descriptions. I imagine it was done for consistency with other solutes, but this raises an interesting pedagogical question. If the chemical formula was intentionally suppressed in the visual design, shouldn't the same apply to the descriptions?

**Comment by emily-phet** [comment]
@zepumph I get where this suggestion comes from, but a chemical formula for Drink Mix is being provided in the descriptions for sticky reasons, and we don't want this propagated to other places in the sim. 

@arouinfar: The chemical formula is provided visually for all of the other solutes - and there's a place in the description where we provide this. It felt 'broken' to not include something in that spot - in descriptions only - for Drink Mix. I agree that for non-sugar free drink mix the main ingredient is sugar, but that also felt odd to say that the main ingredient of drink mix (which could be interpreted as just the *mix*) was something else entirely - sugar. As I mentioned...it's a sticky situation. If we don't put a chemical formula, we have to say something to indicate there isn't just missing information... We could put "chemical formula not shown" but we generally avoid visual-specific language where possible.

Happy to hear any suggestions.

**Comment by emily-phet** [comment]
@twant - also wanted to ask you directly if you had any suggestions for the chemical formula issue? Any ideas, or leanings one way or the other? 

One thing I forgot to mention in the list of reasons for having a chemical formula for Drink Mix in the above comment, is this one (probably the biggest one!):
Since drink mix is the initial solute, it is likely to be the solute selected when students read through the PDOM the first time. If there is no chemical formula there - they may *never* know that this information is available for other solutes. So they won't look there later for that information.

**Comment by arouinfar** [comment]
@emily-phet thanks for the clarification.

> I agree that for non-sugar free drink mix the main ingredient is sugar, but that also felt odd to 
say that the main ingredient of drink mix (which could be interpreted as just the mix) was something else entirely - sugar.

I interpret "Drink Mix" as the powder one mixes with water to prepare a beverage, which would include the sweetener. If citric acid had the same solubility as sucrose for the ranges available in the simulation, I would be fine with using its chemical formula for Drink Mix. However, an aqueous solution of citric acid saturates well below 5.0 M (max in the sim), so the description "Drink Mix contains mainly citric acid" is very problematic. 

> Since drink mix is the initial solute, it is likely to be the solute selected when students read through the PDOM the first time. If there is no chemical formula there - they may never know that this information is available for other solutes. So they won't look there later for that information.

That is indeed a tricky situation, and a reasonable motivation for the differences between the visual design and descriptions.

**Comment by kathy-phet** [comment]
> We could put "chemical formula not shown" but we generally avoid visual-specific language where possible.

What if you said instead "No chemical formula provided". So it does not use visual-related words.

**Comment by twant** [comment]
Thanks for asking, @emily-phet. @arouinfar your points about citric acid are very interesting. It sounds like regardless, just specifying citric acid isn't a great scientific choice. 

I think either specifying citric acid and sucrose or specifying neither (and saying something like @kathy-phet's suggestion) make sense in my mind. I probably lean most strongly toward "No chemical formula provided," particularly because it provides a neater separation between A11y strings and sim strings (if we specify a new chemical formula for drink mix, we now have a chemical formula string that is only being used for a11y descriptions, while all other chemical formula strings are used for both). I wonder if we could say something like "drink mix is composed of multiple chemical compounds, and no formula is provided"

**Comment by kathy-phet** [comment]
> I wonder if we could say something like "drink mix is composed of multiple chemical compounds, and no formula is provided"

I had that thought too. I was wondering if it was going to be too many words to explain it?

**Comment by arouinfar** [comment]
> I wonder if we could say something like "drink mix is composed of multiple chemical compounds, and no formula is provided"

I think that's a really clear way of describing things, and could address the issue @emily-phet raised with students being unlikely to revisit the formula in the PDOM. Unfortunately, it's also a bit verbose, but maybe still acceptable? We could also try tightening it up a bit.

"Drink mix contains multiple chemical compounds, no formula is provided."
"Drink mix contains multiple compounds, no formula is provided."

**Comment by Matthew-phet** [comment]
Hey! thanks everyone for all the input, I talked it over with @emily-phet and we decided to go with 
"Drink mix contains multiple compounds, no chemical formula provided." 

@emily-phet may follow up here later with more information

**Comment by twant** [comment]
@emily-phet @Matthew-phet seems like we've gotten to some resolution on this one. I'm going to close it for now, but please let me know if you'd like to keep it open/discuss further!
